.gitignore

```diff
@@ -1 +1,2 @@
 *.swp
+result
```

Comment:
Nit: result* should catche dev and bin outputs as well, if they land there from other packages.

Reply:
Thanks!